services/networking/loadbalancer/taxonomy.md

```diff
@@ -0,0 +1,67 @@
+| CCC-050339  | Session affinity                                      | Can configure subsequent requests from an initial client to be passed to the same target.                                                                                                                                                                                                                                     |
+| CCC-050340  | Long-lived TCP connection                             | Supports connections that remain open for an extended period without teardown and reestablishment.                                                                                                                                                                                                                            |
+| CCC-050341  | Load Balancing to multiple ports on the same instance | Can load balance to multiple ports on the same instance to distribute incoming traffic across different services or applications running on the same backend server.                                                                                                                                                          |
+| CCC-050342  | Preserve Source IP address                            | Can maintain the original source IP address of incoming requests throughout the load balancing process.                                                                                                                                                                                                                       |
+| CCC-050343  | Websockets                                            | Can handle WebSocket connections.                                                                                                                                                                                                                                                                                             |
+| CCC-050344  | Zonal Isolation                                       | Operation can be restricted to communicate only between resources deployed within a specific availability zone or region.                                                                                                                                                                                                     |
+| CCC-050345  | Regional Load Balancing                               | Load balancers can be distributed across multiple zones in the same region.                                                                                                                                                                                                                                                   |
```

Comment:
Are these two the same? Or is there a slight difference between 'can isolate to a specific zone' and 'can operate in one zone'?

Reply:
CCC-050344 description has the part `or region`. If it is within a region then it becomes CCC-050345. And if `or-region` exist in CCC-050344 it does not match it's name `Zonal Isolation`. Can you check this?

Reply:
True, good point. 44 shouldn't contain 'or region', so I'll remove 'or region' and keep both features.